naas/logger.py

```diff
@@ -0,0 +1,24 @@
+import logging
+
+
+class Logger:
+    DEFAULT_PROGNAME = "NAAS_RUBY_CLIENT"
+
+    def __init__(self):
+        self.logger = logging.getLogger()
+
+    def warn(self, message, progname=DEFAULT_PROGNAME):
+        self.logger.warning(progname + ": " + message)
+
+    def debug(self, message, progname=DEFAULT_PROGNAME):
+        self.logger.debug(progname + ": " + message)
+
+    def fatal(self, message, progname=DEFAULT_PROGNAME):
+        self.logger.critical(progname + ": " + message)
+
+    def error(self, message, progname=DEFAULT_PROGNAME):
+        self.logger.error(progname + ": " + message)
+
+    def info(self, message, progname=DEFAULT_PROGNAME):
+        self.logger.info(progname + ": " + message)
+
```

Comment:
@arthuroe - In the ruby client this is a `Delegate` to the [ruby standard library `Logger`](https://ruby-doc.org/stdlib-2.4.0/libdoc/logger/rdoc/Logger.html). There may be parts of this that don't translate, but I like the direction of you wrapping them in this way. 

They key with these is that we can setup the Logger itself to log to `STDOUT` or log to a specified _file_ at the time of configuration.

Reply:
I have set the logger to log to both a file and to `STDOUT` if the file name to log to is provided in the environment variables. Otherwise, it will just log to `STDOUT`.

Apologies this took long to be responded to. I missed it.

Reply:
@muhallan - code looks good. Waiting to test as my branch is in an unstable state.

Reply:
@muhallan - tested and all looks good!